pkg/schedule/schedulers/hot_region.go

```diff
@@ -1424,6 +1440,8 @@ func (bs *balanceSolver) compareDstStore(detail1, detail2 *statistics.StoreLoadD
 	return 0
 }
 
+// stepRank returns a function can calculate the discretized data,
+// where `rate` will be discretized by `step`.
```

Comment:
Do we need to give more information about "rate" and "step" to explain?

Reply:
added